components/resourceDetails/resourceSharing/addAgentButton/agentPickerModal/index.jsx

```diff
@@ -355,9 +356,21 @@ export default function AgentPickerModal({ type, text, onClose, setLoading }) {
               groups: GROUP_CONTACT,
             }}
           />
-          <AddWebIdButton onClick={handleAddRow} disabled={addingWebId} />
+          <MobileAgentsSearchBar handleFilterChange={handleFilterChange} />
+
+          <AddWebIdButton
```

Comment:
Perhaps more readable/easier to work with to use the Hidden component in Material UI? https://material-ui.com/components/hidden/

Reply:
Means you don't need to introduce CSS classes that does the same (i.e. `mobileOnly` and `desktopOnly`).